task003/main.go

```diff
@@ -0,0 +1,75 @@
+package main
+
+import (
+	"fmt"
+	"sort"
+)
+
+// function that returns an average value of array
+func avgArrays(s [6]int) float64 {
+	var sum float64
+	for _, number := range s {
+		x := float64(number)
+		sum += x
+	}
+	n := sum / float64(len(s))
+	return n
+}
+
+// function that returns the longest word from the slice of strings
+func maxString(words []string) string {
+	var maxWord string
+	for _, word := range words {
+		if len(word) > len(maxWord) {
+			maxWord = word
+		}
+	}
+	return maxWord
+}
+
+// function that returns the copy of the original slice in reverse order
+func reverse(numbers []int64) []int64 {
+	for i := 0; i < len(numbers)/2; i++ {
+		j := len(numbers) - i - 1
+		numbers[i], numbers[j] = numbers[j], numbers[i]
+	}
+	return numbers
+}
```

Comment:
There is a little problem with this code
"Implement reverse([]int64) []int64 function, that returns the **COPY** of the original slice in reverse order. "
But you reverse and return the original slice 
https://play.golang.org/p/e4pgoTKTPbz